Source/WebCore/platform/graphics/filters/software/FELightingSoftwareApplier.cpp

```diff
@@ -29,156 +29,11 @@
 #include "FELightingSoftwareApplier.h"
 
 #include "FELighting.h"
+#include "FELightingSoftwareApplierInlines.h"
 #include "Filter.h"
-#include "ImageBuffer.h"
-#include "PixelBuffer.h"
```

Comment:
I think methods of PixelBuffer are used in FELightingSoftwareApplier::applyPlatform(). So I think "PixelBuffer.h" is needed.

Reply:
When doing the patch I had a local non-unified build and I took out those to see if they were needed. It wouldn't surprise me if those were included somewhere else. Unfortunately we don't have any tooling to tell us whether the includes should be present or not 😞